src/ploneintranet/activitystream/browser/prototype/post.html

```diff
@@ -59,13 +59,34 @@
     <p tal:content="structure view/statusupdate_view/decorated_text">Comment text</p>
   </section>
 
-  <section class="preview">
+  <section class="document preview">
 
     <figure tal:repeat="attachment view/statusupdate_view/attachments">
       <a href="/incredibly-boring-document" tal:attributes="href attachment/link">
         <img src="/media/preview_thumb_1.jpg" alt=""
              tal:attributes="src string:${attachment/img_src}">
       </a>
+          <figcaption>
+            <strong class="title">
+              <a tal:attributes="href attachment/link; alt attachment/alt" href="/projection-material" alt="Projection Material">
+                <tal:r replace="attachment/title">Projection Material</tal:r> <!--span class="page-counter icon-docs">3x</span-->
+              </a>
+              <!-- This needs exposure of all generated thumbs. We can do that and have them accessible, it is just not added here yet. -->
+              <!--span class="functions">
```

Comment:
Why is this not exposed, and what needs to be done for that? Can you maybe add a ticket for that?